### PR TITLE
Fixing schematics dict putput

### PR DESCRIPTION
**Pull request by rszemplinski**

### CODE REVIEW

File: datamodel_code_generator/model/template/schematics/BaseModel.jinja2 [hunk 1]
```diff
@@ -11,9 +11,6 @@ class {{ class_name }}:
     {{ description }}
     """
 {%- endif %}
-{%- if not fields %}
-    pass
-{%- endif %}
```
Comment (pleonowens) [inline datamodel_code_generator/model/template/schematics/BaseModel.jinja2:16]: Why remove this? For classes that don't have fiedlds to just display an empty class body?

File: datamodel_code_generator/model/schematics/schematics_model_field.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+                extra_kwarg_string = ((', ' if inner_type and extra_kwargs else '') + ", ".join(f"{key}={value}"
+                                                                                                for key, value in
+                                                                                                extra_kwargs.items()))
+
+                if is_model_type:
+                    if outer.is_enum:
+                        return f'StringType({extra_kwarg_string})' if is_top_level else f'StringType, {extra_kwarg_string}'
+
+                    if isinstance(outer.reference.source, BaseModel) and outer.reference.source.is_fake_class:
```
Comment (rszemplinski) [inline datamodel_code_generator/model/schematics/schematics_model_field.py:130]: The real changes start here

**Review by pleonowens — APPROVED** [review]

File: datamodel_code_generator/model/template/schematics/BaseModel.jinja2 (hunk 1 above)
Comment (rszemplinski) [reply datamodel_code_generator/model/template/schematics/BaseModel.jinja2:16]: Oops, I removed this to test something out but forgot to put it back in. Nice catch

File: datamodel_code_generator/model/schematics/schematics_model_field.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                        # Using StringType cause there isn't a generic ObjectType
+                        return 'StringType'
```
Comment (pleonowens) [inline datamodel_code_generator/model/schematics/schematics_model_field.py:132]: What about `BaseType`?

File: datamodel_code_generator/model/schematics/schematics_model_field.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+                    if isinstance(outer.reference.source, CustomRootType) and len(outer.reference.source.fields) == 1:
+                        # Using StringType cause there isn't a generic ObjectType
```
Comment (pleonowens) [inline datamodel_code_generator/model/schematics/schematics_model_field.py:135]: Is this comment meant to go here?

**Pull request merged**